packages/css/table.css

```
@@ -38,6 +38,17 @@
     border-bottom: var(--dsc-table-header-divider);
   }
 
+  & > tfoot > tr > :is(th, td) {
+    background: var(--dsc-table-background);
+    border-bottom: none;
+    border-top: var(--dsc-table-border);
+  }
+
+  & caption {
+    @composes ds-heading from './heading.css';
+    text-align: left;
```

Minor pirk if we want to support several text directions in the future ☺️ 
```suggestion
    text-align: inherit;
```